README.md

```diff
@@ -44,6 +44,8 @@ This was originally inspired by [lg](https://github.com/lg)'s [gist](https://gis
 
     The example below enables subnet routing for one subnet, enables use as an exit node (Tailscale 1.6+), and uses a one-off pre-auth key, which can be generated at https://login.tailscale.com/admin/authkeys
 
+    !! Remember to change `192.0.2.0/24` with the subnet(s) you *actually want to expose* to the tailnet.
```

Comment:
As the `!!` won't render to anything that looks particularly good, I'd suggest making this an emoji, e.g. `:information_desk_person:` (💁) or `:warning:` (⚠️). That should be more eye-catching to make sure people do change it.

Reply:
Indeed, it should have been `:bangbang:` (‼️), but the warning icon might be even better. :)  

```suggestion
    :warning: Remember to change `192.0.2.0/24` with the subnet(s) you *actually want to expose* to the tailnet.
```

Reply:
I'm happy to go with any of these :-)